src/textual/widgets/_progress_bar.py

```diff
@@ -71,6 +97,24 @@ def __init__(
         super().__init__(name=name, id=id, classes=classes, disabled=disabled)
         self._start_time = None
         self._percentage = None
+        self.color_scheme = self.validate_color_scheme(color_scheme)
+        self.thickness = self.validate_thickness(thickness)
+
+    def validate_color_scheme(self, color_scheme: BarColorScheme) -> BarColorScheme:
+        if color_scheme is None:
+            color_scheme = _DEFAULT_BAR_COLOR_SCHEME
```

Comment:
So, the idea is that setting to `None` falls back to the default value? (And same thing for the `thickness`?)
If so, I think you should mention that in the docstrings for the reactives.

Reply:
Yes. It is easier to use I guess if we don't have to deal with default constants around the code. If not set Bar will turn None into the default value.